common/command_common.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Let's  create a folder called utils instead of common . so that more utility methods can go in there

Reply:
Suggestion: 
 the file names can be command.py and report.py under utils 

utils/command.py
utils/report.py 

so it is indicating that these are utility classes for command and report

Reply:
Solved in f51f48a838230c01efc75a3d3ba61211ee739821.